### PR TITLE
 add unit tests for security group member hook task 

### PR DESCRIPTION
**Pull request by arvindn05**

### CODE REVIEW

File: osc-server/src/test/java/org/osc/core/broker/service/tasks/conformance/openstack/securitygroup/SecurityGroupMemberHookCheckTaskTestData.java [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+        } catch (Exception e) {
+            throw new RuntimeException();
+        }
+        for (VMPort port : sgm.getVmPorts()) {
+            expectedGraph.appendTask(new SfcFlowClassifierCreateTask().create(sgm.getSecurityGroup(), port));
+        }
+        return expectedGraph;
+    }
+
+    public TaskGraph getSfcWithDeletedPortsTaskGraph(SecurityGroupMember sgm) {
```
Comment (emanoelxavier) [inline osc-server/src/test/java/org/osc/core/broker/service/tasks/conformance/openstack/securitygroup/SecurityGroupMemberHookCheckTaskTestData.java]: can these graph methods be static?

Comment (arvindn05) [reply osc-server/src/test/java/org/osc/core/broker/service/tasks/conformance/openstack/securitygroup/SecurityGroupMemberHookCheckTaskTestData.java]: Thats what i tried initially but the api factory and sdn api being shared was causing interference and test were failing

Comment (emanoelxavier) [reply osc-server/src/test/java/org/osc/core/broker/service/tasks/conformance/openstack/securitygroup/SecurityGroupMemberHookCheckTaskTestData.java]: I see, can you put a TODO on that so we can take a look later? Maybe something we can improve also on the overall test approach.

Comment (arvindn05) [reply osc-server/src/test/java/org/osc/core/broker/service/tasks/conformance/openstack/securitygroup/SecurityGroupMemberHookCheckTaskTestData.java]: will add it as part of next commit

**Pull request merged**